spec/vacationresponse.mdwn

```diff
@@ -69,3 +71,19 @@ The following errors may be returned instead of the *vacationResponseSet* respon
 `accountNoMail`: Returned if the *accountId* given corresponds to a valid account, but does not contain any mail data.
 
 `invalidArguments`: Returned if one of the arguments is of the wrong type, or otherwise invalid (including using an id other than `"singleton"`). A `description` property MAY be present on the response object to help debug with an explanation of what the problem was.
+
+### Special care
```

Comment:
I guess it belongs to "Implementing JMAP" documentation of jmap.io, don't you think ?